docs/install.md

```diff
@@ -29,6 +32,25 @@ import pydantic
 print('compiled:', pydantic.compiled)
 ```
 
+### Performance vs package size trade-off
+
+Compiled binaries can increase the size of your Python environment. If for some reason you want to reduce size the size of your *pydantic* installation you can avoid installing any binaries using the [`pip --no-binary`](https://pip.pypa.io/en/stable/reference/pip_install/#install-no-binary) option. Make sure `cython` is not in your environment, or that you have the `SKIP_CYTHON` environment variable set to avoid re-compiling *pydantic* libraries:
+```
+SKIP_CYTHON=1 pip install \
+  --no-binary pydantic \
+  pydantic
+```
```

Comment:
```bash
    SKIP_CYTHON=1 pip install --no-binary pydantic pydantic
    ```
    !!! note
        `pydantic` is repeated here intentionally - `--no-binary pydantic` tells `pip` you want no binaries for pydantic,
        the next `pydantic` tells `pip` which package to install.

I think this is clearer, people (including me) get confused about the usage of `--on-binary`.

(I can't make this a suggestion as it includes triple back quotes itself)

Reply:
Updated according to suggestion.